src/api/ApiClient.ts

```
@@ -681,10 +682,10 @@ export const addToCart = async (
 
   if (!responseError) {
     setCustomHeader(response)
-    const { attributes } = response?.data?.data
+    const { attributes } = response?.data
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)